docs/static/releasenotes.asciidoc

```diff
@@ -35,25 +35,20 @@ This section summarizes the changes in the following releases:
 
 ==== New features and improvements
 
-===== Expanded JDK ecosystem 
-
-We have improved test scripts infrastructure to accept a specific Java
-Development Kit (JDK) present in the host. The changes include exporting the
-path to a JDK home into an environment variable named BUILD_JAVA_HOME and its
-use by shell scripts to launch test and integration scripts. This work and other
-JDK14 fixes pave the way for Logstash to support both AdoptOpenJDK 11 and 14 in
-the near future . https://github.com/elastic/logstash/pull/11786[#11786],
-https://github.com/elastic/logstash/pull/11839[#11839],
-https://github.com/elastic/logstash/pull/11935[#11935]
-
-===== Expanded JDK platform support 
-
-{ls} runs on many different operating platforms and various flavors of the Java
-Development Kit (JDK), and we’ve expanded this coverage in 7.8.0. {ls} has
-introduced support for running on CentOS/RHEL 8.x and Ubuntu 20.04.
+===== Expanded JDK ecosystem and platform support
+
+We can be more flexible and responsive in supporting new JDKs and deprecating
```

Comment:
LGtm! I would also potentially reference this page https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html